web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
```

Comment:
Still new to the syntax of things, how does this work? This will allow them to try to create the article first, then deny them at this line? (as opposed to just denying them from accessing this controller/view) Am I understanding that correctly?